Makefile

```diff
@@ -1,16 +1,19 @@
 export GOPATH ?= $(shell go env GOPATH)
 export GO111MODULE ?= on
 export GOPRIVATE ?= github.com/insolar
```

Comment:
I think we can delete it, we don't use private repo `insolar/common-test` anymore